### PR TITLE
#132 Added Seeding Script and mongo data pull

### PR DESCRIPTION
**Pull request by sachdevlaksh**
### Fixes Issue
#132 

### Note to reviewers
Kindly review

### CODE REVIEW

**Comment by RakeshSangem** [comment]
Hey @sachdevlaksh, Could you check what causing build failure?


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

File: repo/repo.project.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    return projects;
+});
+
+async function findProjects() : Promise<Project[]> {
+    const prisma = new PrismaClient();
+    return await prisma.project.findMany({take: 30});
+}
+//
+
+
```
Comment (rohitdasu) [inline repo/repo.project.ts:21]: hi @sachdevlaksh just curious to know why did you put this outside of projects page?

File: pages/projects.tsx [hunk 2]
```diff
@@ -9,10 +9,9 @@ import {
   ChildButton,
   Directions,
 } from 'react-floating-button-menu';
-import { projects } from '../sample-data/data';
 import { Icon } from '@iconify/react';
 import 'twin.macro';
-
+import {projects} from "../sample-data/data";
```
Comment (rohitdasu) [inline pages/projects.tsx:14]: i think it's still getting projects from the local json data. could you make it use the mongodb data?

Comment (sachdevlaksh) [reply pages/projects.tsx:14]: Hi @rohitdasu , I want to remove local json data, but I am not able to retrieve mongo data on above file(repo.project.ts) in function(findProjects), but not able to export that data to this file(projects.tsx), Could you please guide how to do that.

Comment (rohitdasu) [reply pages/projects.tsx:14]: sure got it. I will help you. Firstly please pull the latest code from main branch and fix the conflicts.

Comment (sachdevlaksh) [reply pages/projects.tsx:14]: okay I will.

File: repo/repo.project.ts (hunk 1 above)
Comment (sachdevlaksh) [reply repo/repo.project.ts:21]: I am not sure if we can do this on projects page, I thought we are doing db call, so it usually happen on repo layer, so created separate repo module/dir for that

File: pages/projects.tsx [hunk 3]
```diff
@@ -3,17 +3,23 @@ import React, { useState } from 'react';
...
 const Projects = () => {
+  const fetcher = (...args: Parameters<typeof fetch>) =>
+      fetch(...args).then((res) => res.json());
+
+  const { data } = useSWR('/api/project', fetcher)
+  console.log( `SWR data ${data}`);
```
Comment (sachdevlaksh) [inline pages/projects.tsx:22]: Here is the logger. @leewan-09

File: package.json [hunk 4]
```diff
@@ -27,6 +27,7 @@
     "react-floating-button-menu": "^2.1.1",
     "react-hot-toast": "^2.3.0",
     "react-redux": "^8.0.2",
+    "swr": "^1.3.0",
```
Comment (sachdevlaksh) [inline package.json:30]: @leewan-09  Added.

**Comment by rohitdasu** [comment]
@leewan-09 what do you think about the loader screen?

**Comment by rohitdasu** [comment]
can we merge as it is and improve in later PRs? @leewan-09

**Comment by leewan-09** [comment]
Yes, we will improve

**Review by rohitdasu — APPROVED** [review]
LGTM 👍

**Review by leewan-09 — APPROVED** [review]

**Pull request merged**